tests/Utils.ts

```diff
@@ -206,6 +225,16 @@ export class TestAgora implements FullNodeAPI
         );
     }
 
+    /// Implements FullNodeAPI.getMerklePath
+    public getMerklePath (block_height: Height, hash: Hash): Promise<Hash[]>
+    {
+        let sample_merkle_tree = this.blocks[1].merkle_tree;
+
+        return Promise.resolve(
+            [sample_merkle_tree[1], sample_merkle_tree[9], sample_merkle_tree[13]]
```

Comment:
```suggestion
            [
                new Hash(sample_merkle_tree[1]),
                new Hash(sample_merkle_tree[9]),
                new Hash(sample_merkle_tree[13])
            ]
```
The `sample_merkle_tree` is the Array of string